### PR TITLE
Salesforce query - offset added

### PR DESCRIPTION
**Pull request by nedvajz**
Well I know this update is for bit outdated plugin, but hey, maybe somebody still uses like I do for one project ;)

### CODE REVIEW

**Comment by voycey** [comment]
Hey! Thanks!

A few people do still use this, definitely neglected on this side as the project we used this on isn't really happening any longer but ill check this PR out when I get a chance!

File: Salesforce/Model/Datasource/SalesforceSource.php [hunk 1]
```diff
@@ -221,11 +221,7 @@ public function create(Model $model, $fields = null, $values = null) {
          * @return array|mixed
          */
         public function read(Model $model, $queryData = array(), $recursive = null) {
-            $limit = "";
```
Comment (voycey) [inline Salesforce/Model/Datasource/SalesforceSource.php:224]: By removing this how are you handling limits? Old school CakePHP coding I know but at the time Salesforce required a limit or it returned the lot - has this changed?

Comment (nedvajz) [reply Salesforce/Model/Datasource/SalesforceSource.php:224]: The limit is there, just moved on the bottom of method as `LIMIT` is always appended to `$sfq` when it is present. I just get rid of `$limit` variable and append directly (same with `OFFSET`)

Comment (voycey) [reply Salesforce/Model/Datasource/SalesforceSource.php:224]: Ill happily trust you on this - I cant possibly test it due to not having this project running anymore - things look fine :)

Thanks for your contribution - much appreciated :)

**Pull request merged**